src/commands/base/IgnoreBaseCommand.ts

```diff
@@ -0,0 +1,27 @@
+import { Message } from 'discord.js';
+
+import BaseCommand from './BaseCommand';
+import CommandUsage from './CommandUsage';
+
+import DatabaseAdapter from '../../db/DatabaseAdapter';
+
+export default abstract class IgnoreBaseCommand extends BaseCommand implements CommandUsage {
+  public USAGE = '';
```

Comment:
Why does this not have `USAGE` string? If it shouldn't, consider making USAGE optional instead of using empty string.